app/styles/onboarding-card.module.css

```diff
@@ -36,20 +49,21 @@
     font-style: normal;
     font-weight: 700;
     line-height: 24px; 
-    padding-top: 5rem;
+    padding-top: 3rem;
 }
 
-@media (max-width: 800px) {
+@media (max-width: 1024px) {
     .onboarding-card {
-        min-width: 60vh;
+        width: 80%;
+        padding: 2rem;
```

Comment:
Why you added again `padding: 2rem`
already here: https://github.com/Real-Dev-Squad/website-www/pull/648/files#diff-bb9f4c5344f6ff1505af2885cdb84f11eb9ecb1d578d25ee809b6dba3cb75c30R10

Reply:
I haven't add or remove

Reply:
give more context...not understand

Reply:
to make it more systematic I removed old padding and just gave 2rem